docs/mux-logging-service.md

```diff
@@ -49,6 +49,20 @@ There are environment variables for the logging-mux DC config:
 * `FORWARD_LISTEN_HOST` - hostname to listen for secure_forward protocol log
   messages - this is the same as the FQDN in the mux server cert
 
+These are environment variables for the logging-fluentd daemonset config:
+
+* `USE_MUX_CLIENT` - `true`/`false` - default is `false` - if `true`, fluentd
+  will not send logs directly to Elasticsearch, instead it will use
+  `secure_forward` to send logs to mux.
+* `MUX_CLIENT_MODE` - `minimal`/`full_no_k8s_meta` - default is unset - if
+  `minimal`, this is the same behavior as `USE_MUX_CLIENT true`.  If
+  `full_no_k8s_meta`, then fluentd will perform *all* of the data processing
```

Comment:
@richm, certainly that would be possible, but that can also be done via work in 3.7 or 3.6.1 to change that when we get that working properly.  In the mean time, simplifying and streamlining helps us as developers to verify and test this code, and helps the field deploy something that works properly.

Could we just hard code one option for mux in the `full_no_k8s_meta` mode and spend our time verifying that?